.markdownlint.yaml

```diff
@@ -0,0 +1,4 @@
+# Prettier-friendly minimal markdownlint config (turns off rules which prettier handles)
+default: true
```

Comment:
I don't think we want markdown linting in this repository - the only things that are markdown are the readme and the changelog

Reply:
A core value proposition here is to make sure that all your tech is checked, but that's not for everyone. If you don't want to be running tools on all the tech, than no need for a meta-linter :)

Reply:
I think it's more that we want to make sure we know what linters we are running, and to understand them. I don't know what markdownlint is or what it does, and don't know if we want to enforce whatever rules they set.